neo/bin/prompt.py

```diff
@@ -17,6 +17,10 @@
 from prompt_toolkit.token import Token
 from twisted.internet import reactor, task
 
+
+if os.path.exists(os.path.join(os.getcwd(), 'neo')):
```

Comment:
why do we need this -- neo should be already importable, either way? I remember this being an issue way back, before using `pip install -e .`